eox_tagging/api/v1/filters.py

```diff
@@ -26,20 +29,42 @@ class Meta:
         model = Tag
         fields = ["key", "status", "tag_type", "tag_value"]
 
+    @staticmethod
+    def _map_to_model(name, value):
+        """
+        Used to map filter names to model names and their types.
+
+        Example usage:
+            >>> model_field_name, value = self._map_to_model("certificate_id", "1")
+            >>> model_field_name
+            >>> 'id'
+            >>> value
+            >>> 1
+        For now, it just maps certificates.
+        """
+        if name.startswith("certificate_"):
+            name = name[len("certificate_"):]
+            value = int(value) if name == "id" else value
+        return name, value
+
     def filter_by_target_object(self, queryset, name, value):
         """Filter that returns the tags associated with target."""
-        TARGET_TYPES = {
+        target_types = {
             "course_id": "courseoverview",
             "username": "user",
+            "certificate_id": "generatedcertificate",
```

Comment:
I just realized we did something similar with courseenrollment. I'll be changing this implementation to match that one